include/ttmlir/Dialect/TTNN/IR/TTNNOps.td

```diff
@@ -874,12 +875,31 @@ def TTNN_ReduceScatterOp: TTNN_Op<"reduce_scatter"> {
     }];
 
     let arguments = (ins AnyRankedTensor:$input,
+                         TT_Device:$device,
                          SI32Attr:$scatter_split_dim,
-                         TTNN_ReduceType:$math_op,
+                         TT_ReduceTypeAttr:$math_op,
                          DefaultValuedAttr<SI32Attr, "1">:$num_links);
     let results = (outs AnyRankedTensor:$result);
 
     let hasVerifier = 1;
 }
 
+def TTNN_MeshShardOp: TTNN_Op<"mesh_shard"> {
```

Comment:
I couldn't find `mesh_shard` in metal, can you point me to it?

Reply:
@svuckovicTT This is host-side input shard / output concat operation and TTNN team is working on the APIs.